### DISCUSSION TITLE
Windows Authentication in OrchardCore Admin

### DISCUSSION THREAD

**Discussion by Abreu1975** [OrchardCMS/OrchardCore#9955] [Q&A]
Greetings, 

I am relatively new to OrchardCore (created a single decoupled site for real) and dot net core.

My problem is that I must have my site use Windows Authentication, which is quite alright since I am using a decoupled site. Problem is I am using the OrchardAdmin pages and having trouble finding out how to simply use the user from the http request instead of asking for login.

Closer i have got to making it work is making 2 equal websites running on different posts: one with windows authentication, from the frontend, the other with anonymous authentication for the backend.

Is there a straightforward way of just instructing OrchardCore to just use the same user from the http request?

Thank you!

**Comment by hyzx86** [comment]
1. You can implement a custom authentication center website, use a separate site, which will use Windows authentication, used to get user information, we call it WINDOWS Authority Center website
2. Implement an OC's external landing module, and jump to the site mentioned in the first step when the user needs to log in.
3. Generate an Token in the Authorization Center to save the encrypted user login information, passed to the external landing component via the GET.
4. Decrypt information in external landing components, get user information, and automate login and automatic registration

**Comment by hyzx86** [comment]
Sorry, I re-read your question. According to my experience, I have tried similar practices in ABP, and it turns out that it is not feasible. Because if you use a site of Windows authentication, the identity provider will think that the user has logged and it will not trigger the login check, But the user does not have the login operation of the OC, so the user information cannot be obtained.
Maybe my trial is not enough, if there is a better way, please share it, thank you
